bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -1643,4 +1643,24 @@ private static boolean areEntriesOfSegmentStoredInTheBookie(LedgerMetadata ledge
         }
         return firstStoredEntryId != LedgerHandle.INVALID_ENTRY_ID;
     }
+
+    /**
+     * returns boolean value specifying if the ensemble of the segment is
+     * adhering to the ensemble placement policy for the given writeQuorumSize
+     * and ackQuorumSize.
+     *
+     * @param ensembleList
+     *            ensemble of the segment
```

Comment:
ensembleList confused me as if it is a list of ensembles. May be you should consider renaming it to ensembleBookiesList or something

Reply:
changed